aikau/src/main/resources/alfresco/core/WidgetsOverrideMixin.js

```diff
@@ -0,0 +1,261 @@
+       * [
+       *   {
+       *     id: "ADDRESS",
+       *     targetPosition: "END",
+       *     name: "alfresco/forms/controls/TextBox",
+       *     config: {
+       *       fieldId: "ADDRESS",
+       *       label: "Address",
+       *       description: "Where do you live?",
+       *       name: "address"
+       *     }
+       *   }
+       * ]
+       *
+       * @example <caption>Insert a widget before another widget using by using a <b>targetPosition</b> of <b>"BEFORE"</b> and providing a <b>targetId</b></caption>
+       * [
+       *   {
+       *     id: "ADDRESS",
+       *     targetId: "WIDGET_2"
+       *     targetPosition: "BEFORE",
+       *     name: "alfresco/forms/controls/TextBox",
+       *     config: {
+       *       fieldId: "ADDRESS",
+       *       label: "Address",
+       *       description: "Where do you live?",
+       *       name: "address"
+       *     }
+       *   }
+       * ]
+       *
+       * @example <caption>Insert a widget after another widget using by using a <b>targetPosition</b> of <b>"AFTER"</b> and providing a <b>targetId</b></caption>
+       * [
+       *   {
+       *     id: "ADDRESS",
+       *     targetId: "WIDGET_1"
+       *     targetPosition: "AFTER",
+       *     name: "alfresco/forms/controls/TextBox",
+       *     config: {
+       *       fieldId: "ADDRESS",
+       *       label: "Address",
+       *       description: "Where do you live?",
+       *       name: "address"
+       *     }
+       *   }
+       * ]
+       *
+       * @example <caption>Remove a widget using using the <b>remove</b> attribute with the <b>id</b> of the widget to remove.</caption>
+       * [
+       *   {
+       *     id: "WIDGET_2",
+       *     remove: true
+       *   }
+       * ]
+       *
+       * @example <caption>Replace a widget using the <b>replace</b> attribute with the <b>id</b> of the widget to replace.</caption>
+       * [
+       *   {
+       *     id: "WIDGET_2",
+       *     replace: true,
+       *     name: "alfresco/forms/controls/TextArea",
+       *     config: {
+       *       fieldId: "ADDRESS",
+       *       label: "Location",
+       *       name: "address"
+       *     }
+       *   }
+       * ]
+       *
+       * @example <caption>Merge new configuration into an existing widget <b>id</b></caption>
+       * [
+       *   {
+       *     id: "WIDGET_1",
+       *     name: "alfresco/forms/controls/TextArea",
+       *     config: {
+       *       name: "Who are you?"
+       *     }
+       *   }
+       * ]
+       *
+       * @instance
+       * @param {object[]} widgets The default widgets
+       * @param {object[]} overrides The overrides to apply
+       */
+      applyWidgetOverrides: function alfresco_core_WidgetsOverrideMixin__applyWidgetOverrides(widgets, overrides) {
+         var existingWidgetIndex;
+
+         if (ObjectTypeUtils.isArray(widgets) && ObjectTypeUtils.isArray(overrides))
```

Comment:
Worth adding a log message in the else block rather than just silently failing if a object is provided?